tests/integration/test_tickets.py

```diff
@@ -141,6 +141,17 @@ def test_tickets_DELETE(client, auth_headers):
     assert is_valid(response, 404)
     assert response.json == {"message": "Ticket not found"}
 
+    ticketsToDelete = [create_ticket().id, create_ticket().id]
+    deleteIds = {"ids": ticketsToDelete}
+    response = client.delete(endpoint, json=deleteIds, headers=auth_headers["pm"])
+    assert is_valid(response, 200)
+    assert response.json == {"message": "Tickets successfully deleted"}
+
+    missingIds = {"notIds": [5]}
+    response = client.delete(endpoint, json=missingIds, headers=auth_headers["pm"])
+    assert is_valid(response, 400)
+    assert response.json == {"message": "Ticket IDs missing in request"}
+
```

Comment:
Can we put these tests into a different function? It makes it difficult to read here since they're testing a different endpoint then when we're deleting a single resource.